docs/sphinx/user_guide/tutorial/view_layout.rst

```diff
@@ -55,20 +55,31 @@ which we define as:
    :language: C++
 
 Here we define three ``RAJA::View`` objects, 'Aview', 'Bview', and 'Cview',
-that *wrap* the array data pointers, 'A', 'B', and 'C', respectively. We 
-pass a data pointer as the first argument to each view constructor and then 
+that *wrap* the array data pointers, 'A', 'B', and 'C', respectively. We
+pass a data pointer as the first argument to each view constructor and then
 the extent of each matrix dimension as the second and third arguments. There
 are two extent arguments since we indicate in the ``RAJA::Layout`` template
-parameter list. The matrices are square and each extent is 'N'. Here, the 
-template parameters to ``RAJA::View`` are the array data type 'double' and 
+parameter list. The matrices are square and each extent is 'N'. Here, the
+template parameters to ``RAJA::View`` are the array data type 'double' and
 a ``RAJA::Layout`` type. Specifically::
 
   RAJA::Layout<2, int>
 
-means that each View represents a two-dimensional default data layout, and 
-that we will use values of type 'int' to index into the arrays. 
+means that each View represents a two-dimensional default data layout, and
+that we will use values of type 'int' to index into the arrays.
 
-Using the ``RAJA::View`` objects, we can access the data entries for the rows 
+.. note:: A third argument in the Layout type can be used to specify the index
+          with unit stride::
+
+            RAJA::Layout<2, int, 1>
+
+          In the example above index 1 will be marked to have unit stride making
```

Comment:
The note is correct in that it correctly identifies the stride-1 index in the 2-dim'l layout case. The default stride-1 index in our convention is the rightmost index. I would suggestion adding that the third argument is optional.  Also, I would use a permuted layout example to illustrate the optimized use case to avoid extraneous indexing arithmetic.

Reply:
Resolved as we discussed at the meeting that the default is  -1 marking no index as containing unit stride.